angrmanagement/plugins/log_human_activities/log_human_activities.py

```diff
@@ -0,0 +1,136 @@
+import logging
+import hashlib
+from ..base_plugin import BasePlugin
+
+l = logging.getLogger(__name__)
+l.setLevel('INFO')
+
+try:
+    from slacrs import Slacrs
+    from slacrs.model import HumanActivityVariableRename, HumanActivityFunctionRename, HumanActivityClickBlock, HumanActivityClickInsn, HumanActivityCommentChanged, HumanActivityRaiseView
+except ImportError as ex:
+    Slacrs = None  # type: Optional[type]
+    HumanActivityVariableRename = None  # type: Optional[type]
+    HumanActivityFunctionRename = None  # type: Optional[type]
+
+
+# TODO: add created_by field
+TODO = "TODO"
+
+
+class LogHumanActivitiesPlugin(BasePlugin):
+    def __init__(self, *args, **kwargs):
+        if not Slacrs:
+            raise Exception("Skipping LogHumanActivities Plugin. Please install Slacrs to Initialize it.")
+        super().__init__(*args, **kwargs)
+        self.session = Slacrs().session()
```

Comment:
Note to myself: I added a CHECRS-backend-selection plugin. We should just use the backend string there to initialize Slacrs.